assets/icons/conditional-content-icon.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * WordPress dependencies
+ */
+import { Path, SVG } from '@wordpress/components';
+
+const restrictedContent = (
+	<SVG viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+		<Path d="M5.5 18L5.5 6C5.5 5.72386 5.72386 5.5 6 5.5H10.5V4L6 4C4.89543 4 4 4.89543 4 6L4 18C4 19.1046 4.89543 20 6 20H10.5V18.5H6C5.72386 18.5 5.5 18.2761 5.5 18Z" />
+		<Path d="M16.5 4V5.5L15 5.5L15 4H16.5Z" />
+		<Path d="M20 6C20 4.89543 19.1046 4 18 4V5.5C18.2761 5.5 18.5 5.72386 18.5 6H20Z" />
+		<Path d="M18.5 16.5L18.5 14.5H20V16.5H18.5Z" />
+		<Path d="M20 13H18.5V11H20V13Z" />
+		<Path d="M18 18.5C18.2761 18.5 18.5 18.2761 18.5 18H20C20 19.1046 19.1046 20 18 20V18.5Z" />
+		<Path d="M16.5 20V18.5H15V20H16.5Z" />
+		<Path d="M18.5 9.5H20V7.5H18.5V9.5Z" />
+		<Path d="M13.5 4V5.5L12 5.5L12 4H13.5Z" />
+		<Path d="M13.5 18.5V20H12L12 18.5H13.5Z" />
+	</SVG>
```

Comment:
Just ran it through an optimizer:

```suggestion
	<SVG viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
		<Path d="M5.5 18V6a.5.5 0 01.5-.5h4.5V4H6a2 2 0 00-2 2v12a2 2 0 002 2h4.5v-1.5H6a.5.5 0 01-.5-.5zm11-14v1.5H15V4h1.5zM20 6a2 2 0 00-2-2v1.5a.5.5 0 01.5.5H20zm-1.5 10.5v-2H20v2h-1.5zM20 13h-1.5v-2H20v2zm-2 5.5a.5.5 0 00.5-.5H20a2 2 0 01-2 2v-1.5zM16.5 20v-1.5H15V20h1.5zm2-10.5H20v-2h-1.5v2zm-5-5.5v1.5H12V4h1.5zm0 14.5V20H12v-1.5h1.5z"/>
	</SVG>
```

Reply:
Cool! Which one did you use?

Reply:
This was the `Advanced SVG Export` plugin in Figma. I've also found Inkscape good for stroke -> path conversion.
Sorry about the lint issue :)